wafer/users/templates/wafer.users/registration_confirm_mail.txt

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% blocktrans %}
+Thank you for registering for {{ conference_name }}.
```

Comment:
That really should be mentioned in a comment then, because just using a different name is not a great hint for people looking at things later.